R/pkg/R/group.R

```diff
@@ -199,17 +199,10 @@ createMethods()
 #' Applies a R function to each group in the input GroupedData
 #'
 #' @param x a GroupedData
-#' @param func A function to be applied to each group partition specified by GroupedData.
-#'             The function `func` takes as argument a key - grouping columns and
-#'             a data frame - a local R data.frame.
-#'             The output of `func` is a local R data.frame.
-#' @param schema The schema of the resulting SparkDataFrame after the function is applied.
-#'               The schema must match to output of `func`. It has to be defined for each
-#'               output column with preferred output column name and corresponding data type.
-#' @return a SparkDataFrame
 #' @rdname gapply
 #' @name gapply
 #' @export
+#' @seealso \link{gapplyCollect}
 #' @examples
 #' \dontrun{
```

Comment:
Similar to the params being duplicated, right now the examples from the DataFrame and GroupedData versions are duplicated in the generated Rd file. I think this is fine in general, but in the case of `gapply` and `dapply` we can unify the examples and just show that we can call `gapply` with or without a `group_by` ?